website_event_baidu_map/website_event_baidu_map.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <data>
+        <template id="event_description_full_inherited" name="map" inherit_id="website_event.event_description_full">
+            <xpath expr="//div[@class='panel-body']/a" position="replace">
+                <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+                <meta name="viewport" content="initial-scale=1.0, user-scalable=no" />
+                <style type="text/css">
+                    body, html {width: 100%;height: 100%;margin:0;font-family:"微软雅黑";}
+                    #allmap{width:100%;height:250px;}
+                    p{margin-left:5px; font-size:14px;}
+                </style>
+                <script type="text/javascript" t-att-src="request.registry['ir.config_parameter'].get_param(request.cr, request.uid, 'baidu.api')"></script>
+                <div id="allmap"></div>
+                <script type="text/javascript">
+                    // BaiDu map API v2.0 function
+                    var map = new BMap.Map("allmap"); 
+                    map.addControl(new BMap.MapTypeControl());    
+                    map.centerAndZoom(new BMap.Point(121.480237, 31.236305), 11);
```

Comment:
Somehow true but not sure if we can avoid it (the map needs to be started with a location).